src/envoy/mixer/http_control.h

```diff
@@ -37,7 +37,8 @@ typedef std::shared_ptr<HttpRequestData> HttpRequestDataPtr;
 class HttpControl final : public Logger::Loggable<Logger::Id::http> {
  public:
   // The constructor.
-  HttpControl(const std::string& mixer_server);
+  HttpControl(const std::string& mixer_server,
+              const std::map<std::string, std::string>& attributes);
```

Comment:
use 

std::map<std::string, std::string>&& attributes

call it by std::move()

to reduce a copy